repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -91,7 +91,14 @@ class ExecutorClassLoader(conf: SparkConf, classUri: String, parent: ClassLoader
       inputStream.close()
       Some(defineClass(name, bytes, 0, bytes.length))
     } catch {
-      case e: Exception => None
+      case e: FileNotFoundException =>
+        // We did not find the class
+        logDebug(s"Did not load class $name from REPL class server at $uri", e)
```

Comment:
This happens whenever a class is loaded which was not defined in the REPL (Spark Shell). But it could theoretically be an error (e.g. whole directory not available). The class server is the server that serves the classes (i.e. the actual `.class` files) defined in the REPL.